src/Midi_hook.cpp

```diff
@@ -29,6 +29,7 @@ MidiHook::MidiHook(const QString &json_string)
 	value_as_filter = obs_data_get_bool(data, "value_as_filter");
 	value.emplace(obs_data_get_int(data, "value"));
 	setAction();
+	initHotkey();
```

Comment:
why is this being done here?

Reply:
Ah... I see my mistake. I wanted to initialize OBS Hotkey binding on hook creation, but this constructor is only used on config load, where hotkeys are not loaded yet.
I will delete it